extension/src/test/suite/experiments/index.test.ts

```diff
@@ -1470,6 +1470,8 @@ suite('Experiments Test Suite', () => {
         rows: [
```

Comment:
It really fails here, but when diffing both, they're identical. Not sure what's happening and I spent way too much time on this.
<img width="1628" alt="Screenshot 2023-05-31 at 2 32 59 PM" src="https://github.com/iterative/vscode-dvc/assets/3683420/9c093da7-87cf-40fe-954f-1e4ab57c0632">